optuna_dashboard/_app.py

```diff
@@ -449,6 +452,44 @@ def save_trial_note(study_id: int, trial_id: int) -> dict[str, Any]:
         response.status = 204  # No content
         return {}
 
+    @app.get("/csv/<study_id:int>")
+    def download_csv(study_id: int) -> BottleViewReturn:
+        # Create a CSV file
+        summary = get_study_summary(storage, study_id)
+        if summary is None:
+            response.status = 404  # Not found
+            return {"reason": f"study_id={study_id} is not found"}
+        trials = get_trials(storage, study_id)
+
```

Comment:
An error happens when study has no trial
```suggestion
        if len(trials) == 0:
            return {"reason": f"study_id={study_id} has no trials"}

```

Reply:
In this case, the downloaded csv file is as follows. Is this intended?

<img width="438" alt="Screenshot 2023-12-06 at 16 39 52" src="https://github.com/optuna/optuna-dashboard/assets/38826298/64b9422c-46d9-4a7e-8b28-df14dd40567a">

Reply:
I just confirmed that we can remove this line after addressing three of my last comments, which are already committed.
So we can remove this line.

Reply:
Finally, I obtained `study` and `study_name` from storage. I stopped using `summary`.